src/Engine/Utils.java

```diff
@@ -508,6 +508,7 @@ public static void sortLocationsByTravelTime(Unit unit, ArrayList<XYCoord> mapLo
     TravelDistanceComparator tdc = new TravelDistanceComparator(unit, map);
     Collections.sort(mapLocations, tdc);
   }
+  
```

Comment:
Aaaaagh. You can't just go writing viruses into our game like this. People will notice.

Reply:
Viruses?